frontend/tests/e2e-tests.sh

```
@@ -200,9 +226,16 @@ else
 fi
 echo "=========================================================================================="
 
-if [[ " ${SCRIPT_SHORT_ARGS[@]} " =~ " -q " ]]; then
-	echo "| quick mode"
-	npx playwright test ./tests/functional/"${TEST_PATHS[@]}" -x --project=chromium "${SCRIPT_LONG_ARGS[@]}"
+FRONTEND_HASH_FILE="$APP_DIR/frontend/tests/.frontend_hash"
+FRONTEND_HASH=$(find "$APP_DIR/frontend/src" -type f \( -name "*.ts" -o -name "*.svelte" \) | xargs -I {} md5sum {} | md5sum)
```

Shellcheck (0.10.0)

[warning] Use 'find .. -print0 | xargs -0 ..' or 'find .. -exec .. +' to allow non-alphanumeric filenames.

(SC2038)